conda/environments/runtime_cuda-125_arch-x86_64.yaml

```diff
@@ -49,8 +48,10 @@ dependencies:
 - watchdog=3.0
 - websockets
 - pip:
+  - --extra-index-url https://download.pytorch.org/whl/cu124
```

Comment:
what is the old dependency this is replacing?

Reply:
we're getting `torch` from pypi instead of conda because conda does not have the version we need.